rest-service/manager_rest/test/endpoints/test_deployment_updates.py

```diff
@@ -123,6 +124,89 @@ def test_remove_node_and_relationship(self):
             self.assertEquals(1 if param in changed_params else 0,
                               len(execution.parameters[param]))
 
+    def test_plugin_installation_updates_plugins_by_default(self):
+        def assert_all_not_in(plugin_names, plugins_set):
+            for plugin_name in plugin_names:
+                self.assertNotIn(plugin_name, plugins_set)
+
+        def assert_all_in(plugin_names, plugins_set):
```

Comment:
There is an assert that asserts lists without order

Reply:
There's `assertItemsEqual` but that checks for equality and I need to assert it's a sublist of the other.

Reply:
Are you sure about the less restrict 'in'? can it be more strict like startswith?